ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -27,14 +27,23 @@ public final class Atomic<Value> {
 	/// Initializes the variable with the given initial value.
 	public init(_ value: Value) {
 		_value = value
+		let result = pthread_mutex_init(&mutex, nil)
+		assert(0 == result, "Failed to initialize mutex with error \(result).")
```

Comment:
Haha I guess I think it's more readable that way – "assert 0" is the essence of the statement. I'll switch it up.